Task3.js

```diff
@@ -0,0 +1,27 @@
+//Conditional Statement and Data Types
+// Выполнить сложение различных типов(string+boolean, string+number, number+boolean)
+// Выполнить умножение различных типов(string * boolean, string * number, number * boolean)
+// Выполнить деление различных типов(string/boolean, string/number, number/Boolean)
+let str = 'hello world', num = 5, bool = true;
```

Comment:
1. Use `const` if variable will not be re-assigned.
2. Use one const or let declaration per variable or assignment (https://github.com/airbnb/javascript#variables).